pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -152,10 +169,7 @@ func (ic *ifConfigurator) configureContainerLink(
 	ifaceIdx := 1
 	containerIP.Interface = &ifaceIdx
 
-	ifaceName := fmt.Sprintf("vEthernet (%s)", endpoint.Name)
-	if err := util.SetInterfaceMTU(ifaceName, mtu); err != nil {
-		return fmt.Errorf("failed to configure MTU on container interface '%s': %v", ifaceName, err)
-	}
+	ic.addAsyncFuncRequest(epName, ic.configureMTU, mtu)
```

Comment:
Please add comments to explain, why it needs to be async.

Reply:
Added.